hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -53,6 +55,12 @@
       default: "24"
       local:   "true"
     }
+    { name:    "NumLocality"
+      desc:    "The number of locality TPM module supports."
```

Comment:
does this refer to the local direct response TPM registers?

Reply:
The usages of `NumLocality` are:

- Instantiate TPM_ACCESS_x registers
- Check Out of Locality access and return Invalid response by HW